test/catalyst/gemsCatalystsRegistry.ts

```diff
@@ -0,0 +1,326 @@
+import {ethers, getNamedAccounts} from 'hardhat';
+import {BigNumber} from '@ethersproject/bignumber';
+import {expect} from '../chai-setup';
+import {setupGemsAndCatalysts} from './fixtures';
+describe('GemsCatalystsRegistry', function () {
+  it('getMaxGems for catalystId = 1 should be 1', async function () {
+    const {
+      gemsCatalystsRegistry,
+      commonCatalyst,
+    } = await setupGemsAndCatalysts();
+    const catalystId = await commonCatalyst.catalystId();
+    const maxGems = await gemsCatalystsRegistry.getMaxGems(catalystId);
+    expect(maxGems).to.equal(1);
+  });
+
+  it('getMaxGems for non existing catalystId should fail', async function () {
+    const {gemsCatalystsRegistry} = await setupGemsAndCatalysts();
+    expect(gemsCatalystsRegistry.getMaxGems(10)).to.be.revertedWith(
+      'CATALYST_DOES_NOT_EXIST'
+    );
+  });
+
+  it('burnCatalyst should burn 2 common catalysts from catalystOwner account', async function () {
+    const {
+      gemsCatalystsRegistry,
+      commonCatalyst,
+    } = await setupGemsAndCatalysts();
+    const {catalystOwner} = await getNamedAccounts();
```

Comment:
instead of getting the `catalystOwner` from named accounts you would get it from the fixture